tests/test_simulator.py

```diff
@@ -15,7 +15,7 @@ def test_no_navmesh_smoke():
     # with no navmesh and the navmesh isn't used for any exisitng sensors
     agent_config.sensor_specifications = []
 
-    sim_cfg.scene_id = "data/scene_datasets/habitat-test-scenes/stage_floor1.glb"
+    sim_cfg.scene_id = "data/test_assets/scenes/stage_floor1.glb"
```

Comment:
Can we have assert in simulator.py that the scene_id exists?

Reply:
Something along these lines obviously need to be explored as scene import is constructed.

Reply:
This really feels like an important blocker because segfaulting when someone makes a typo is not desirable behavior.